### PR TITLE
Scheduler alternatives

### PR DESCRIPTION
**Pull request by TheoEb**
Related to https://github.com/Scalingo/documentation/pull/1392 and [CRON-92]

[CRON-92]: https://scalingo.atlassian.net/browse/CRON-92?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: _posts/platform/app/task-scheduling/2000-01-01-scalingo-scheduler.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+{% endwarning %}
+
+Scalingo Scheduler is here to help you run scheduled tasks at regular intervals.
+
+The syntax used to describe this interval is the same as in the cron software.
+
+Scalingo Scheduler launches tasks as [one-off containers]({% post_url platform/app/2000-01-01-tasks %}) in detached mode. Therefore the related one-off documentation and their detached mode applies.
+
+{% warning %}
+Scheduled tasks execution is expected but not guaranteed. Scalingo Scheduler is known to occasionally (but rarely) miss the execution of scheduled jobs. If scheduled tasks are a critical component of your application, it is recommended to [run a custom clock process]() instead for more reliability, control, and visibility.
```
Comment (yannski) [inline _posts/platform/app/task-scheduling/2000-01-01-scalingo-scheduler.md:20]: @TheoEb add the link to the custom clock process doc page

File: _posts/platform/app/task-scheduling/2000-01-01-scalingo-scheduler.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  "jobs": [
+    {
+      "command": "*/10 * * * * rails orders:check",
+      "size": "2XL"
+    }
+  ]
+}
+```
+
+**Your cron expression must be set with at least a 10 minutes interval.**
```
Comment (yannski) [inline _posts/platform/app/task-scheduling/2000-01-01-scalingo-scheduler.md:54]:
```suggestion
{% warning %}
Your cron expression must be set with at least a 10 minutes interval.
{% endwarning %}
```

File: _posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
+---
+title: Custom Clock Processes
+nav: Custom Clock Processes
```
Comment (EtienneM) [inline _posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md:3]: nitpick: if I remember well, if the `nav` is the same as the `title` it is not useful to add it again
```suggestion
title: Custom Clock Processes
```

File: _posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+index: 2
+---
+
+With custom clock processes, you have the ability to specify custom schedule. You will also have other benefits like
+environment parity between development and production, or time precision in your task scheduling.
+
+## Definition of the Custom Clock
+
+The implementation of the custom clock will vary depending on the used language. However, defining the clock process
+is very simple and is done in the `Procfile`.
```
Comment (EtienneM) [inline _posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md:15]: As specified in our documentation (https://www.notion.so/scalingooriginal/Writing-Documentation-at-Scalingo-bb7985d6ea16455894417e36ede09cea#3b845d6a0b3f45f8808a90a3f650f65f), the words like "simple" shouldn't be used in documentation
```suggestion
 is done in the `Procfile`.
```

File: _posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+
+#### Example 
+
+Here is an example of the process definition using the Ruby library`clockwork`:
```
Comment (EtienneM) [inline _posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md:19]:
```suggestion
Here is an example of the process definition using the Ruby library `clockwork`:

```

File: _posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+```yaml
+web: bundle exec puma -t 1:3 -p $PORT
```
Comment (EtienneM) [inline _posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md:21]: nitpick: I would use the same puma command as we already do in the doc (https://doc.scalingo.com/languages/ruby/start#using-puma-as-a-production-server)

```suggestion
web: bundle exec puma -C config/puma.rb
```

File: _posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+otherwise each framework has its own task scheduler. You may want to use:
+
+* [Laravel scheduler]({% post_url languages/php/2000-01-01-laravel %}#laravel-tasks-scheduler)
+* [cron/cron](https://packagist.org/packages/cron/cron)
+* [liebig/cron](https://packagist.org/packages/liebig/cron)
+* [heartsentwined/zf2-cron](https://packagist.org/packages/heartsentwined/zf2-cron)
+
+#### Example
+
+A complete example project can be found [here](https://github.com/Scalingo/sample-php-cron)
```
Comment (EtienneM) [inline _posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md:92]:
```suggestion
A complete example project can be found [here](https://github.com/Scalingo/sample-php-cron).
```

File: _posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md [hunk 8]
```diff
@@ -0,0 +1,193 @@
...
+  },
+  start: true,
+  timeZone: 'Europe/Paris'
+});
+
+console.log('Started 2 cron jobs')
+```
+
+With this example, the `job1` ticks every 2 minutes and the `job2` ticks every minute. Each job
+display a message in the log of the application.
```
Comment (EtienneM) [inline _posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md:191]:
```suggestion
displays a message in the log of the application.
```

File: _posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md [hunk 9]
```diff
@@ -0,0 +1,194 @@
...
+
+With custom clock processes, you have the ability to specify custom schedule. You will also have other benefits like
+environment parity between development and production, or time precision in your task scheduling.
+
+## Definition of the Custom Clock
+
+The implementation of the custom clock will vary depending on the used language. However, defining the clock process
+is very simple and is done in the `Procfile`.
+
+#### Example
```
Comment (yannski) [inline _posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md:17]:
```suggestion
### Example 
```

File: _posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md [hunk 10]
```diff
@@ -0,0 +1,194 @@
...
+## Start the Task Scheduler
+
+Once your application has been deployed, scale your 'clock' to 1 to start the task
+scheduler:
+
+```bash
+$ scalingo --app my-app scale clock:1
+```
+
+## Implementation
```
Comment (yannski) [inline _posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md:35]:
```suggestion
## Implementation examples
```

File: _posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md [hunk 11]
```diff
@@ -0,0 +1,194 @@
...
+### Ruby
+
+In Ruby you can use [clockwork](http://rubygems.org/gems/clockwork),
+[resque-scheduler](https://rubygems.org/gems/resque-scheduler) or
+[sidekiq-scheduler](https://rubygems.org/gems/sidekiq-scheduler) (Sidekiq
+Enterprise has cron-like feature built-in) for example.
+
+#### Example
+
+The following example uses the `clockwork` package.
```
Comment (yannski) [inline _posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md:46]:
```suggestion
```

**Pull request merged**